docs/src/upgrade/release-3.8.x.asciidoc

```diff
@@ -1347,6 +1347,10 @@ to continue to refine this `Channelizer` implementation and it can be deprecated
 
 See: link:https://issues.apache.org/jira/browse/TINKERPOP-3168[TINKERPOP-3168]
 
+==== Runtime Upgrades
+
+Gremlin Go has been upgraded to Go version 1.25.
```

Comment:
Combine this with https://github.com/apache/tinkerpop/blob/3.8-dev/docs/src/upgrade/release-3.8.x.asciidoc?plain=1#L983 so that the two runtime changes are in the same place